src/midi-router.js

```diff
@@ -9,7 +9,7 @@
 "use strict";
 
 var fluid = require("infusion"),
-    flock = fluid.require("flocking");
+    flock = fluid.require("flocking"); // eslint-disable-line no-unused-vars
```

Comment:
But if it's only used via IoC, then surely linting won't be concerned with it, right? You only refer to <code>flock</code> within strings in this file. So why declare the variable? The namespace, if I am not too terribly confused, will still be visible to Infusion by virtue of simply requiring the module.